ipa-core/src/protocol/basics/reveal.rs

```diff
@@ -32,6 +27,16 @@ pub trait Reveal<C: Context, B: RecordBinding>: Sized {
     async fn reveal<'fut>(&self, ctx: C, record_binding: B) -> Result<Self::Output, Error>
     where
         C: 'fut;
+
+    /// partial reveal protocol to open a shared secret to all helpers except helper `left_out` inside the MPC ring.
+    async fn partial_reveal<'fut>(
```

Comment:
`PartialReveal` trait is removed too, I don't see a why `partial_reveal` should be on its own trait.